misaligned.js

```diff
@@ -1,16 +1,31 @@
-const {expect} = require('chai')
+const {expect} = require('chai');
+const {assert} = require('chai');
+const {MAJOR_COLOR_NAMES, MINOR_COLOR_NAMES} = require("./colorConstants");
 
-function print_color_map() {
-    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+function get_color_map(majorColors,minorColors) {
+    let colorMap = [];
     for (let i = 0; i < majorColors.length; i++) {
-        for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);

```

Comment:
5 is a magic number. The calculation is now clear, but sometime in future if the number of minor colors change then this formula too changes. Please add a test case to check the boundary, for example pair 1 , pair 5, pair 6. This will make the production code robust against unintentional changes due to extension in future.